src/main/java/me/botsko/prism/Prism.java

```diff
@@ -331,9 +331,11 @@ public void onEnable() {
             ActionMeter.setupActionMeter(monitor.getRegistry());
         }
         log("Initializing Prism " + pluginVersion + ". Originally by Viveleroi; maintained by the AddstarMC Network");
-        PaperLib.suggestPaper(this);
-        // Load configuration, or install if new
         loadConfig();
+        // Load configuration, or install if new
+        if (!getConfig().getBoolean("prism.suppress-Paper-message",false)) {
```

Comment:
I'd suggest lowercasing the `Paper` in that option, just to keep it consistent

Reply:
and, just to be pedantic, insert a space after the comma